javav2/example_code/acmpca/src/main/java/com/example/acmpca/CreateCertificateAuthority.java

```diff
@@ -21,12 +21,12 @@
 
 // snippet-start:[acmpca.java2.CreateCertificateAuthority.main]
 /**
- * Before running this Java V2 code example, set up your development environment, including your
- * credentials.
- *
- * <p>For more information, see the following documentation topic:
- *
- * <p>https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/get-started.html
+ * Before running this Java V2 code example, set up your development 
+ * environment, including your credentials.
+ * <p>
```

Comment:
looking at the other samples, I think we can actually just eliminate these  `<p>`s